### PR TITLE
Enable format before saving

### PR DESCRIPTION
**Pull request by chansuke**
Add format before saving.

### CODE REVIEW

**Comment by splattael** [comment]
Wow nice, thank you! :green_heart:

:+1: from me

@lucerion WDYT?

File: plugin/rufo.vim [hunk 1]
```diff
@@ -12,13 +16,22 @@ if !exists('g:rufo_errors_buffer_position')
   let g:rufo_errors_buffer_position = 'bottom'
 endif
 
+if get(g:, "rufo_saving", 0)
+  autocmd BufWritePre *.rb silent! call rufo#format_file()
```
Comment (lucerion) [inline plugin/rufo.vim]: @chansuke could you, please, explain why it uses here? Looks like it do the same thing as `RufoFormatBeforeSave`

Comment (chansuke) [reply plugin/rufo.vim]: @lucerion Thank you for the review.I forgot to delete the draft code.
Firstable I wrote it and then updated to `RufoFormatBeforeSave`.
I will delete this line.

**Comment by lucerion** [comment]
@chansuke thank you!

**Comment by splattael** [comment]
@lucerion @chansuke :purple_heart:

**Pull request merged**